modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="enabling-assisted-installer-service-on-bare-metal_{context}"]
+
+.Enabling Assisted Installer Service on Bare Metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. {rh-rhacm-first} ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the {rh-rhacm} hub cluster.
+
+For distributed units DUs, {rh-rhacm} supports single node OpenShift deployments. Single node OpenShift runs on a single bare metal host. When the host is provisoned, the single node cluster acts as both supervisor and worker node. {rh-rhacm} supports single node OpenShift deployments for Shift Container Platform {product-title} on top of a single BMO For distributed units (DUs). Single node OpenShift runs on a single bare-metal host. Once provisoned, the single node cluster acts as both control plane and worker node.
```

Comment:
First instance of DU is at the beginning of the paragraph. Parens should be there. 

This still sounds odd: {rh-rhacm} supports single node OpenShift deployments for Shift Container Platform {product-title} on top of a single BMO For distributed units (DUs).

What is Shift Container Platform? With {product-title}, that's going to render as "Shift Container Platform OpenShift Container Platform".

Reply:
"Single node OpenShift runs on a single bare-metal host." is a repeat of an earlier sentence. You can delete that.

Reply:
Updated as suggested.